### PR TITLE
CREATE/CREATE2 a contract with all possible first bytes

### PR DESCRIPTION
**Pull request by gumb0**
Pulled out of #835 removing London expectations, should be useful on its own to check that deploying all undefined bytes is allowed in Berlin.

### CODE REVIEW

File: src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  expect:
+    - indexes:
+       data: !!int -1
+       gas: !!int -1
+       value: !!int -1
+      network:
+        - 'Berlin'
+      result:
+       a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml:54]: can add a few addresses with deployed code here to show and to see that accounts with that bytecode has actually been deployed

File: src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      balance: 100000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul
+        {
+          // { mstore8(0, 0x00) return(0, 1) }
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml]: this is the contract constructor code.
so you need actually to return the bytecode of a contract that is being created. here it is 0x00

or do you try to create2 a contract with constructor code with invalid code before return?

Comment (gumb0) [reply src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml]: Yes, intention is to create contracts with code that consists of one byre

File: src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml (hunk 1 above)
Comment (gumb0) [reply src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml:54]: Added

File: src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml (hunk 2 above)
Comment (gumb0) [reply src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml]: Added some comments in yul code.

**Pull request merged**